lib/hyacinth/adapters/term_search_adapter/solr.rb

```diff
@@ -35,7 +35,7 @@ def find(vocabulary, uri)
         end
 
         # Solr query. Returns solr json.
-        def search
+        def search(post: false)
```

Comment:
I wonder if it would be better to just post 100% of the time if we know we'll need to sometimes?  I don't know if there's an advantage to doing `get` sometimes and `post` other times.

Reply:
I was wondering about this actually. That way we don't have to worry about specifying it in certain places.

Reply:
Yeah, I think that would be the best way to go then.

Reply:
Will do. I realized that because we add the projects filter automatically the searches are pretty long.

Reply:
👍